src/styles/blocks/nav.css

```diff
@@ -0,0 +1,16 @@
+.nav__list {
```

Comment:
The task requires you to write styles in the 'main.scss' file, but the current file is a CSS file. Consider moving these styles to 'main.scss' and use SCSS syntax.